CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#9478](https://github.com/rubocop-hq/rubocop/pull/9478): Add new `Style/LegacyHash` cop. ([@zverok][])
+
```

Comment:
Please refer [this][1] for creating changelog entries. It will create separate files under `/changelog` which are then merged into this main changelog during release.

[1]: https://github.com/rubocop-hq/rubocop/blob/master/CONTRIBUTING.md#creating-changelog-entries